lib/app_profiler/profile_id.rb

```diff
@@ -1,19 +1,39 @@
 # frozen_string_literal: true
 
-require "active_support/current_attributes"
 require "securerandom"
 
 module AppProfiler
   class ProfileId
-    class Current < ActiveSupport::CurrentAttributes
-      attribute :id
+    PROFILE_ID_KEY = :__app_profiler_profile_id__
```

Comment:
This should be nested in `Current`, right? It is only used there.

Reply:
https://github.com/Shopify/app_profiler/pull/159/commits/1694438963e7c07a70921d1936e69fd345ac11bc#diff-d41d7a8a119f9102743b4fd40a1caffd15cfc79fc3f3ad23ceceea40d2e56d4bR8